app/(admin)/admin/notifications/page.tsx

```diff
@@ -1,16 +1,72 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import { JSX } from 'react';
+'use client';
+import { Button } from '@/components/Button/Button';
+import { getCurrentLeague } from '@/api/apiFunctions';
+import { Input } from '@/components/Input/Input';
+import { JSX, useState } from 'react';
+import { Label } from '@/components/Label/Label';
+import { sendEmailNotifications } from './actions/sendEmailNotification';
+import React from 'react';
 
 /**
  * The admin home page.
  * @returns The rendered AdminHome page.
  */
 const AdminNotifications = (): JSX.Element => {
+  const [content, setContent] = useState<string>('');
+  const [sendEmailUsers, setSendEmailUsers] = useState<string[]>([]);
+  const [subject, setSubject] = useState<string>('');
+
+  /**
+   * To grab all users from the league.
+   */
+  const participantsEmail = async (): Promise<void> => {
+    try {
+      const leagueId = '66f1a8e300102bff03ff';
+      const leagueData = await getCurrentLeague(leagueId);
+      setSendEmailUsers(leagueData.participants);
+    } catch (error) {
+      console.error('Error Sending Email:', error);
```

Comment:
Remove console error, as throw new Error can be stacked and takes care of this, as discussed in previous PR's from Shashi